src/cli.js

```diff
@@ -1,3 +1,6 @@
+// Disable warnings about template strings in this file, as we do lots of
+// string concatenation so options/help in the CLI app display nicely.
+// jscs:disable requireTemplateStrings
```

Comment:
Yeah sounds good - I prefer the latter too. I wanted to use template strings for this but for multi-line strings whitespace handling would result in something resembling PHP's heredoc. We could of course write a tag to handle that in time.